Vagrantfile

```diff
@@ -55,4 +55,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         config.hostmanager.manage_host = true
         config.hostmanager.aliases = settings['sites'].map { |site| site['map'] }
     end
+
+    # Solr
+    config.vm.network :forwarded_port, guest: 8983, host: 8983
```

Comment:
oh! this makes sense. thanks for adding it

Reply:
I got tired of having to add the port every time I rebooted the VM :)